app/pages/patient/patientinfo.js

```diff
@@ -374,12 +391,12 @@ var PatientInfo = React.createClass({
     }
 
     var birthday = formValues.birthday;
-    if (!(birthday && datetimeUtils.isValidDate(birthday))) {
+    if (!(birthday && sundial.isValidDate(birthday))) {
```

Comment:
This doesn't work quite as intended either.

It allows you to save a completely invalid date without exposing any error:
![screenshot 2015-03-08 14 06 48](https://cloud.githubusercontent.com/assets/1588547/6547684/6aa4e3c4-c59c-11e4-815c-3af10d4a98d6.png)

But then silently translates it into a valid date:
![screenshot 2015-03-08 14 06 55](https://cloud.githubusercontent.com/assets/1588547/6547685/70b9fc0e-c59c-11e4-9666-7fbfcc364a44.png)

I don't know if there's anything we can do about this - I think it's how moment works :/